sdk/formrecognizer/Azure.AI.FormRecognizer/api/Azure.AI.FormRecognizer.netstandard2.0.cs

```diff
@@ -344,6 +344,7 @@ public partial class CopyModelOperation : Azure.Operation<Azure.AI.FormRecognize
     public partial class DocTypeInfo
     {
         internal DocTypeInfo() { }
+        public Azure.AI.FormRecognizer.DocumentAnalysis.DocumentBuildMode? BuildMode { get { throw null; } }
```

Comment:
Not sure if I missed a discussion about this, but should `BuildMode` be required? From my understanding it's present in every scenario (build, copy, compose).

Reply:
was it marked as required in the swagger? it does seem like it should be present

Reply:
It's not ([swagger here](https://github.com/Azure/azure-rest-api-specs/blob/8e73bccfad3f6873b9f91d3920ce0f96af554378/specification/cognitiveservices/data-plane/FormRecognizer/preview/2022-01-30-preview/FormRecognizer.json#L1078)). We can ask the service team about it for the next release.